test/integration/config/ldp-with-auth.json

```diff
@@ -34,20 +34,60 @@
             ]
           }
         },
-        {
-          "RecordObject:_record_key": "handler",
-          "RecordObject:_record_value": { "@id": "urn:solid-server:default:LdpHandler" }
-        },
         {
           "RecordObject:_record_key": "store",
           "RecordObject:_record_value": { "@id": "urn:solid-server:default:ResourceStore" }
+        },
+        {
+          "RecordObject:_record_key": "factory",
+          "RecordObject:_record_value": { "@id": "urn:solid-server:default:ServerFactory" }
```

Comment:
```suggestion
          "_record_value": { "@id": "urn:solid-server:default:ServerFactory" }
```
etc.?

Reply:
This doesn't work yet for key/value objects.

Reply:
> This doesn't work yet for key/value objects.

Yet... (see my comment in https://github.com/solid/community-server/pull/709)

> There are also a couple places where this compaction can not be done yet (such as ConstantMetadataWriter:_headers_key) due to a bug in the generator. I will look into this at a later point in time via a separate PR.